src/Avalonia.Controls/Platform/ExportAvaloniaModuleAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Avalonia.Platform
+{
+    [AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+    public class ExportAvaloniaModuleAttribute : Attribute
+    {
+        public ExportAvaloniaModuleAttribute(string name, Type moduleType)
+        {
+            Name = name;
+            ModuleType = moduleType;
+        }
+
+        public string Name { get; private set; }
+        public Type ModuleType { get; private set; }
+
+        public string RequiredWindowingSubsystem { get; set; } = "";
```

Comment:
So I think this needs clarification in doc comments. That is: what happens if you have 2 attributes, one with a subsystem specified and one without? Are both called when the subsystem matches?